apps/chat/src/store/models/models.epics.ts

```diff
@@ -129,6 +132,51 @@ const getModelsEpic: AppEpic = (action$, state$) =>
     }),
   );
 
+const getInstalledModelIdsEpic: AppEpic = (action$, state$) =>
+  action$.pipe(
+    filter(ModelsActions.getInstalledModelIds.match),
+    withLatestFrom(state$),
+    switchMap(() => {
+      return ClientDataService.getInstalledDeployments().pipe(
+        switchMap((installedModels) => {
+          if (!installedModels?.length) {
+            return of(ModelsActions.getInstalledModelIdsFail());
+          }
+          return of(ModelsActions.getInstalledModelsSuccess(installedModels));
+        }),
+        catchError(() => {
+          return of(ModelsActions.getInstalledModelIdsFail());
+        }),
+      );
+    }),
+  );
+
+const getInstalledModelIdsFailEpic: AppEpic = (action$, state$) =>
+  action$.pipe(
+    filter(ModelsActions.getInstalledModelIdsFail.match),
+    withLatestFrom(state$),
+    switchMap(() => {
+      const defaultModelIds = SettingsSelectors.selectDefaultRecentModelsIds(
+        state$.value,
+      );
+      return of(ModelsActions.updateInstalledModelIds(defaultModelIds));
+    }),
+  );
+
+const updateInstalledModelIdsEpic: AppEpic = (action$) =>
```

Comment:
```suggestion
const updateInstalledDeploymentIdsEpic: AppEpic = (action$) =>
```

I think we should use or Deployments or models

Reply:
In our code we are using term `models` (all neighboring fields)